interface/forms/eye_mag/report.php

```diff
@@ -607,11 +606,16 @@ function narrative($pid, $encounter, $cols, $form_id, $choice = 'full')
               <td><?php echo text($GLAREOSVA); ?></td>
             </tr>
             <?php } if ($CONTRASTODVA||$CONTRASTOSVA) { ?>
-            <tr>
-              <td><?php echo xlt('Contrast{{Constrast Visual Acuity}}'); ?></td>
-              <td><?php echo text($CONTRASTODVA); ?></td>
-              <td><?php echo text($CONTRASTOSVA); ?></td>
-            </tr>
+                <tr>
+                    <td><?php echo xlt('Contrast{{Constrast Visual Acuity}}'); ?></td>
+                    <td><?php echo text($CONTRASTODVA); ?></td>
+                    <td><?php echo text($CONTRASTOSVA); ?></td>
+                </tr>
+            <?php } if (!empty($BINOCVA)) { ?>
+                <tr>
+                  <td><?php echo xlt('VABiNoc{{|Binocular Visual Acuity}}'); ?></td>
```

Comment:
Just checking if you meant to have that `|` character in there to the left of the Binocular.

Reply:
No.  Will fix this